src/wallet/qi-hdwallet.ts

```diff
@@ -69,31 +70,27 @@ export class QiHDWallet extends HDWallet {
         return neuteredAddressInfo;
     }
 
-    // getNextChangeAddress(account: number, zone: string): NeuteredAddressInfo {
-    // 	return this._getNextAddress(account, zone, this._changeAddresses, true);
-    // }
-
-    importOutpoints(outpoints: OutpointInfo[]): void {
-        this._outpoints.push(...outpoints);
-    }
-
-    getOutpoints(zone: string): OutpointInfo[] {
-        if (!this.validateZone(zone)) throw new Error(`Invalid zone: ${zone}`);
-        return this._outpoints.filter((outpoint) => outpoint.zone === zone);
-    }
-
-    /**
-     * Signs a Qi transaction and returns the serialized transaction
-     *
-     * @param {QiTransactionRequest} tx - The transaction to sign.
-     *
-     * @returns {Promise<string>} The serialized transaction.
-     * @throws {Error} If the UTXO transaction is invalid.
-     */
-    async signTransaction(tx: QiTransactionRequest): Promise<string> {
-        const txobj = QiTransaction.from(<TransactionLike>tx);
-        if (!txobj.txInputs || txobj.txInputs.length == 0 || !txobj.txOutputs)
-            throw new Error('Invalid UTXO transaction, missing inputs or outputs');
+	public importOutpoints(outpoints: OutpointInfo[]): void {
```

Comment:
Update zone string on `OutpointInfo` type and validate `OutpointInfo[]` values are valid